cms/djangoapps/contentstore/course_info_model.py

```diff
@@ -60,17 +60,21 @@ def update_course_updates(location, update, passed_id=None, user=None):
 
     if passed_id is not None:
         passed_index = _get_index(passed_id)
-        # oldest update at start of list
-        if 0 < passed_index <= len(course_update_items):
-            course_update_dict = course_update_items[passed_index - 1]
-            course_update_dict["date"] = update["date"]
-            course_update_dict["content"] = update["content"]
-            course_update_items[passed_index - 1] = course_update_dict
-        else:
+
```

Comment:
Define `course_update_dict` and set it to `None` because you are using it at L#70. is it already defined?

Reply:
Done.